### ISSUE TITLE
 elevated plasma proinsulin and pro-opiomelanocortin (POMC) 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9796]
**Preferred term label:**
 elevated plasma proinsulin and pro-opiomelanocortin
(POMC) 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
600955 
PMID: 9207799   1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000419
- label: Elevated circulating proinsulin concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:9207799
- def: The concentration of proinsulin in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9796
- synonyms: Elevated plasma proinsulin(NONE)
